packages/commonwealth/client/scripts/views/components/Profile/ProfileActivity/ProfileActivityRow/ProfileActivityRow.tsx

```diff
@@ -158,31 +158,36 @@ const ProfileActivityRow = ({ activity }: ProfileActivityRowProps) => {
                 iconLeft: 'linkPhosphor',
                 iconLeftSize: 'regular',
                 label: 'Copy link',
-                onClick: async () => {
-                  if (isThread) {
-                    await navigator.clipboard.writeText(
-                      `${domain}/${communityId}/discussion/${id}`,
-                    );
-                    return;
-                  }
-                  await navigator.clipboard.writeText(
-                    `${domain}/${communityId}/discussion/${comment.thread?.id}?comment=${comment.id}`,
-                  );
+                onClick: () => {
+                  const handleAsync = async () => {
+                    if (isThread) {
+                      await navigator.clipboard
+                        .writeText(`${domain}/${communityId}/discussion/${id}`)
+                        .catch(console.error);
```

Comment:
we have saveToClipboard

Reply:
merged master just now and this seems to have been replaced with a newer component in another PR.